### ISSUE TITLE
Logit adjustment loss 

### ISSUE THREAD

**Issue by caide199212** [Chumsy0725/logit-adj-pytorch#6]
For the way 2 using logit adjustment loss, the output logits for inference accuracy in the validation don't perform the logits adjustment, but  the validation loss use the logits adjustment.
As shown in the snippet below:
```
elif args.logit_adj_train:
     loss = criterion(output + args.logit_adjustments, target_var)
acc = utils.accuracy(output.data, target)
```

**Comment by shadowamy** [comment]
It also runs like this in Google's source code(https://github.com/google-research/google-research/blob/master/logit_adjustment/main.py), where train and val are only adjusted for the calculation of loss.
